scripts/mood.coffee

```diff
@@ -0,0 +1,19 @@
+# Description:
+#   Hubot will flip a fucking coin.
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   mood - declare your daily mood
```

Comment:
This seems like it tells you what ur mood is, not u declaring what ur mood is

Reply:
!?!??!!

Reply:
It is declaring your mood, but I can adjust language if wanted